### DISCUSSION TITLE
Add "Last updated" timestamp to the statuses.

### DISCUSSION THREAD

**Discussion by Andre601** [upptime/upptime#557] [Ideas]
While the github action updates the status every 5 minutes can people who visit the status page not know about this, nor how old the currently displayed status is.

Because of this should the different status reports have a "Last updated" timestamp that either displays the date or the relative time since the last update (if doable).
When a date would be displayed should there also be a setting to allow us to change the date format because not everyone would f.e. use `MM-dd-yyyy` in their country or language.

**Comment by pi-nftfi** [comment]
+1
Can I add a 'Last updated' timestamp to my status page and README?

**Reply by AnandChowdhary** [reply]
On the README, you can proxy the "last updated" using the last commit date, but it's a great addition for the status page! :)

**Reply by Andre601** [reply]
I assume changes would need to be made to the status page repository?

If so, does it use a library already that would allow easy date formatting?

**Reply by AnandChowdhary** [reply]
Yes, and there's already a PR open for adding a last updated. I don't think we need a library, we can do `.toLocaleString()` in the browser. :)
